bundles/http_cache.rst

```diff
@@ -108,24 +108,17 @@ Symfony Http Cache
 
 The Symfony HTTP cache is the default caching client for Sulu. It is integrated directly into Sulu.
 
-It works by "wrapping" the kernel. You can find it in the website front controller ``web/website.php``:
+It works by "wrapping" the kernel. This is done by the kernel itself, but in the front controller
```

Comment:
i think there is a negation missing?

Reply:
Why? :thinking: I don't think so, although it is possible that the sentence could be rewritten to be easier to read. But I am currently not changing too much :see_no_evil:

Reply:
@nnatter front controller = index.php. Front controller is the single entry point to a webapplicaiton, not be confused with symfony controllers